src/gui/owncloudgui.cpp

```diff
@@ -1019,7 +1019,7 @@ void ownCloudGui::slotShowShareDialog(const QString &sharePath, const QString &l
     bool resharingAllowed = true; // lets assume the good
     if (fileRecord.isValid()) {
         // check the permission: Is resharing allowed?
-        if (!fileRecord._remotePerm.contains('R')) {
+        if (!fileRecord._remotePerm.isNull() && !fileRecord._remotePerm.hasPermission(RemotePermissions::CanReshare)) {
```

Comment:
Sorry to open that again, but this is a behavior change, right? Previously empty permissions would not allow resharing, and now they do?

Reply:
Yes, it is a behavior change, but that is a fix.  If the server don't have permissions (old server) then we should allow resharing.

If the server give empty permissions, isNull will be false.